tests/main/xdg-open-portal/task.yaml

```diff
@@ -0,0 +1,102 @@
+summary: Verify xdg-open forwarding requests to the desktop portal
+
+# Only enable the test on systems we know portals to function on.
+# Expand as needed.
+systems:
+  - debian-sid-*
+  - fedora-*
+  - opensuse-*
+  - ubuntu-18.04-*
+  - ubuntu-19.10-*
+  - ubuntu-20.04-*
```

Comment:
I've enabled Arch Linux now based on your feedback.  I don't think xdg-desktop-portal-gtk is installed on the systems where the test is passing.

I've updated the desktop portal test setup to move any `*.portal` files under `/usr/share/xdg-desktop-portal/portals` out of the way, and to move them back on restore.

I'd hoped this might solve the problem for Debian 9, but that doesn't seem to be the case.